cfme/cloud/keypairs.py

```diff
@@ -0,0 +1,34 @@
+
+import cfme.fixtures.pytest_selenium as sel
+from cfme.web_ui import Quadicon, Region, flash
+from utils.pretty import Pretty
+from functools import partial
+from cfme.web_ui import toolbar as tb
+from utils.wait import wait_for
+
+
+details_page = Region(infoblock_type='detail')
+cfg_btn = partial(tb.select, "Configuration")
+
+
+class KeyPairs(Pretty):
+    """ docstring for KeyPairs """
+
+    def __init__(self, name=None):
+        self.name = name
+
+    def delete(self):
+        sel.force_navigate('clouds_key_pairs', context={'key pairs': self})
+        cfg_btn("Remove this Keypairs from the VMDB", invokes_alert=True)
+        sel.handle_alert()
```

Comment:
Methods that also handle alerts inside get the `cancel=False` as a param and then they pass it through to the handle_alert.